data/remnant ships.txt

```diff
@@ -35,6 +36,9 @@ ship "Starling"
 		"cloak" .02
 		"cloaking energy" 5
 		"cloaking fuel" .1
+		"outfit scan power" 12
+		"outfit scan speed" 1
+		"tactical scan power" 26
```

Comment:
Seeing as how the scanning tech is now inbuilt, one of the ships should mention in their description that Remnant all ships have in built outfit and tactical scanners.

Reply:
The description has been added to the Albatross:

```"	Spurred on by their exposure to alien ruins, the Remnant have ensured that every ship has detailed scanners built into their hull. While they continue to hope to find wrecks to accompany the ruins, these scanners have proven invaluable in their pursuit of livelier ships."```